source/agora/test/GossipProtocol.d

```diff
@@ -0,0 +1,286 @@
+/*******************************************************************************
+
+    Contains tests for Gossip Protocol.
+
+    Copyright:
+        Copyright (c) 2019 BOS Platform Foundation Korea
+        All rights reserved.
+
+    License:
+        MIT License. See LICENSE for details.
+
+*******************************************************************************/
+
+module agora.test.GossipProtocol;
+
+import agora.common.Data;
+
+import agora.protocol.GossipProtocol;
+
+import std.digest.sha;  
+ 
+ /// Test API
+interface TestAPI {
```

Comment:
I would remove `TestAPI` and `TestRemoteNode` and simply go with the `RemoteAPI!(API)` tests.

Reply:
@Geod24
Check Please ~